templates/internal/appName/rpc/rpc.go.tpl

```diff
@@ -181,8 +181,8 @@ func (gs *GRPCService) StartServers(ctx context.Context, servers *Servers, opts.
 	// end gRPC RPCs injected by modules
 	{{- end }}
 
-	// Register default server
-	api.Register{{ title $pkgName }}Server(s, rpcserver{servers.DefaultServer})
+	// Register default server, title function won't work well when use underscore, so we make it dash first
+	api.Register{{ $pkgName | replace "_" "-" | title | replace "-" "" }}Server(s, rpcserver{servers.DefaultServer})
```

Comment:
Use `goPackageSafeName` function here instead?

Reply:
It won't work, pkgName is already from it.
```{{- $pkgName := stencil.ApplyTemplate "goPackageSafeName" }}```

Reply:
Ah, I see